src/register_target.cpp

```diff
@@ -21,22 +21,42 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+#include <string>
 #include <unordered_map>
 #include <migraphx/register_target.hpp>
+#include <migraphx/ranges.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+std::unordered_map<std::string, dynamic_loader>& target_lib_loader_map()
```

Comment:
This function should be `void store_target_lib(dynamic_loader lib)`. And the global variable should not be exposed outside of the function.

Reply:
Done.